### ISSUE TITLE
Concatenative wiki needs to rel="nofollow" all links in wiki texts

### ISSUE THREAD

**Issue by mrjbq7** [factor/factor#844]
Too much spam.

**Comment by erg** [comment]
The machinery is in ``farkup`` but we need to use it somehow. Links to within the wiki itself don't matter, but outgoing links could be gamed. Nobody seems to be doing this lately.

```factor
t link-no-follow? [ "[[http://lol.com/search?q=sex|haha]]" convert-farkup ] with-variable
```

**Comment by mrjbq7** [comment]
Fixed in 38a6417b70f12e5294b33cd2a43d665b0fe51caf.
